cmd/storage.go

```diff
@@ -61,9 +61,9 @@ var storageUnmountCmd = &cobra.Command{
   odo storage unmount dbstorage
 
   # Unmount storage 'database' from component 'mongodb'
-  odo storage unmount database --component mongodb
-`,
-	Args: cobra.ExactArgs(1),
+  odo storage unmount database --component mongodb`,
```

Comment:
The "," in then end, should be in the next line for consistency with the other commands